### ISSUE TITLE
SrtpCHannel::unprotectRtp what warning is that?

### ISSUE THREAD

**Issue by francoisTemasys** [ging/licode#242]
This message happen many time, sometime with a visible impact on the quality of stream (freezes), sometimes not.
```
2015-01-28 11:08:27,996  - WARN: SrtpChannel - Error SrtpChannel::unprotectRtp 9 packettype 244 pt 116
```
Is it an expected warning? What is it caused by?

**Comment by jnoring** [comment]
I don't think I've ever seen that--what versions of openssl are you using?  And are their specific browsers or combinations of browser that reproduce it?  (and is this the trickle_ice branch, or current master?)

**Comment by IVBakker** [comment]
Here are the dependencies I use:
```
LIBAV_VERSION="libav-11"
LIBNICE_VERSION="libnice-0.1.7"
OPENSSL_VERSION="openssl-1.0.1i"
SRTP_VERSION="1.4.2"
OPUS_VERSION="opus-1.1"
```

I don't have specific browser. It happened with Chrome. It happened on master before and now ice trickle. No specific way to reproduce that's ennoying.
Lastly it happened less.

Surprising that you never saw it.

**Comment by lodoyun** [comment]
That means the replay check for strip fails because of the sequence number of the packet. 
You are using a different version of libsrtp than the one we include in Licode (1.4.4). We did some minor modifications to libsrtp, that must be the reason you are having those errors.

**Comment by IVBakker** [comment]
I understand. I'll need to update. Just to be sure of what you said, has the change you made officially been released to version 1.4.4 or it's a homemade patch?

**Comment by francoisTemasys** [comment]
I actually made a mistake on the version I was using. I'm actually following the master branch and I'm on this commit: bdac15654fd7c05c4116f246c05bf49b6c795b2c .
